cmd/bucket-notification-utils.go

```diff
@@ -274,6 +278,8 @@ func unmarshalSqsARN(queueARN string) (mSqs arnSQS) {
 	switch sqsType {
 	case queueTypeAMQP:
 		mSqs.Type = queueTypeAMQP
+	case queueTypeMQTT:
```

Comment:
Can you add a test case for `unmarshalSqsARN`  TestUnmarshalSQSARN? - Thanks @splinter98